packages/realm/src/Collection.ts

```diff
@@ -16,17 +16,13 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+import type { Dictionary, List, Results } from "./internal";
 import { CallbackAdder, IllegalConstructorError, Listeners, TypeAssertionError, assert, binding } from "./internal";
 
 /**
- * Abstract base class containing methods shared by Realm **List**, **Dictionary**, and **Results**.
+ * Abstract base class containing methods shared by Realm {@link List}, {@link Dictionary} and {@link Results}.
  *
- * A Realm Collection is a homogenous sequence of values of any of the types
```

Comment:
I think we should keep the snippets on how it is similar to a JS Array

Reply:
It got moved to the `OrderedCollection` as this statement is no longer valid for `Collection`.